packages/xod-client/src/core/styles/components/ReactDatasheet.scss

```
@@ -0,0 +1,97 @@
+span.data-grid-container, span.data-grid-container:focus {
+  outline: none;
+}
+
+.data-grid-container .data-grid {
+  table-layout: fixed;
+  border-collapse: collapse;
+}
+
+.data-grid-container .data-grid .cell.updated {
+    background-color: rgba(0, 145, 253, 0.16);
+    transition: background-color 0ms ease ;
+}
+
+.data-grid-container .data-grid .cell {
+  box-sizing: border-box;
+  user-select: none;
+  cursor: cell;
+  background-color: $dark-light;
+  color: $input-color-text;
+  transition: background-color 100ms ease;
+  vertical-align: middle;
+  text-align: right;
+  border: 2px solid $chrome-bg;
+
+  padding: 5px 4px;
+}
+
+.data-grid-container .data-grid .cell.selected {
+  border: 2px double $selected;
+  transition: none;
+  box-shadow: inset 0 -100px 0 rgba(33, 133, 208, 0.15);
+}
+
+.data-grid-container .data-grid .cell.read-only {
+  background: whitesmoke;
+  color: #999;
+  text-align: center;
+}
+
+.data-grid-container .data-grid .cell > .value-viewer {
+  width: 140px;
+  height: 16px;
+  overflow-x: visible;
+}
+
+.data-grid-container .data-grid .cell > input {
+  outline: none !important;
+  border: 0;
+  text-align: right;
+  color: $input-color-text;
+  background: none;
+  display: block;
+  height: 14px;
+  width: 138px;
+}
+
+
+.data-grid-container .data-grid .cell {
+  vertical-align: bottom;
+}
+
+.data-grid-container .data-grid .cell,
+.data-grid-container .data-grid.wrap .cell,
+.data-grid-container .data-grid.wrap .cell.wrap,
+.data-grid-container .data-grid .cell.wrap,
+.data-grid-container .data-grid.nowrap .cell.wrap,
+.data-grid-container .data-grid.clip .cell.wrap {
+  white-space: normal;
+}
+
+.data-grid-container .data-grid.nowrap .cell,
+.data-grid-container .data-grid.nowrap .cell.nowrap,
+.data-grid-container .data-grid .cell.nowrap,
+.data-grid-container .data-grid.wrap .cell.nowrap,
+.data-grid-container .data-grid.clip .cell.nowrap {
+  white-space: nowrap;
+  overflow-x: visible;
+}
+
+.data-grid-container .data-grid.clip .cell,
+.data-grid-container .data-grid.clip .cell.clip,
+.data-grid-container .data-grid .cell.clip,
+.data-grid-container .data-grid.wrap .cell.clip,
+.data-grid-container .data-grid.nowrap .cell.clip {
+  white-space: nowrap;
+  overflow-x: hidden;
+}
+
+.data-grid-container .data-grid .cell .value-viewer, .data-grid-container .data-grid .cell .data-editor {
+  display: block;
+}
+
+.data-grid-container tr:first-child .cell {
+  font-weight: bold;
+  text-align: center;
+}
```

```suggestion
  background-color: $grey;
}
```
It's a little lighter than other cells.